pkg/test/framework/components/environment/kube/cluster.go

```diff
@@ -27,17 +27,15 @@ var _ resource.Cluster = Cluster{}
 // Cluster for a Kubernetes cluster. Provides access via a kube.Accessor.
 type Cluster struct {
 	*kube.Accessor
-	filename            string
-	index               resource.ClusterIndex
-	controlPlaneCluster bool
```

Comment:
Setting this on the `Cluster`s makes it difficult to override by modifying the `ControlPlaneTopology` in setup.

Reply:
https://github.com/istio/istio/pull/23344